.cspell.json

```diff
@@ -17,7 +17,15 @@
         ".database",
         "database",
         ".api",
-        "**logs"
+        "**logs",
+        "**/src/api/test-mocks/mock-application.ts",
+        "**/src/ui/server/test-mocks/mock-application.ts",
+        "**/server/test-mocks/mock-sic-codes.ts",
+        "**/src/api/test-mocks/mock-declarations.ts",
+        "**/src/ui/server/test-mocks/mock-declarations.ts",
+        "**/src/ui/server/test-mocks/mock-applications.ts",
+        "**/server/test-mocks/mock-account.ts",
+        "**/src/api/test-mocks/mock-buyer.ts"
```

Comment:
What's the reason to ignore specific test mock files?

Reply:
These files have various mock application IDs which are being ignored.